pkgs/lapack.yaml

```diff
@@ -33,3 +33,7 @@ build_stages:
 - name: setup_builddir
   mode: override
   bash: |
+
+when_build_dependency:
+- {set: 'LAPACK_LDFLAGS', value: '-Wl,-rpath,${ARTIFACT}/lib -L${ARTIFACT}/lib -llapack'}
```

Comment:
Yes, `-Wl,rpath,` works on both Linux and OS X, while `-Wl,rpath=` does not work on OS X.